docs/releases/v0.11.0.rst

```diff
@@ -157,6 +157,8 @@
     previously it could return a dict of Frames if multiple sources were
     detected. Use :func:`iread()` if you need to read multi-source input.
 
+  -[api] Function: The behaviour of DT.sort() is made consistent with
+    DT[:, :, sort()] when the list of columns to sort is empty.
```

Comment:
Guess it is better to say

```
The behaviour of  :meth:`.sort()`  is made consistent with the :func:`sort()` clause. When the list of columns to sort is empty, both will not sort any columns.
```

Note the usage of the `:meth` and `func:`, this will automatically create links to documentation.

Reply:
Thanks very much.